<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDynamicWave</class>
 <widget class="QMainWindow" name="frmDynamicWave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>467</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SWMM Dynamic Wave Options</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="cmdOK">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdCancel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInertial">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inertial Terms</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboInertial">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblNormal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal Flow Criterion</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboNormal">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblForce">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Force Main Equation</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboForce">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxUseVariable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Variable Time Steps</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAdjusted">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adjusted By</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPercent">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbxAdjusted">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblMinimum">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Variable Time Step (sec)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMinimum">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTimeStep">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Step for Conduit Lengthening (sec)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLengthening">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSurface">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Nodal Surface Area (sq feet)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSurfaceArea">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblMaximum">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum Trials Per Time Step</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbxTrials">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblHead">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Head Convergence Tolerance (feet)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblThreads">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Threads</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboThreads">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTolerance">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
